db_scripts/mosip_certify/dml/certify-key_policy_def.csv

```diff
@@ -2,3 +2,4 @@ app_id,key_validity_duration,pre_expire_days,access_allowed,is_active,cr_by,cr_d
 ROOT,2920,1125,NA,TRUE,mosipadmin,now()
 CERTIFY_SERVICE,1095,60,NA,TRUE,mosipadmin,now()
 CERTIFY_PARTNER,1095,60,NA,TRUE,mosipadmin,now()
+CERTIFY_MOCK_RSA,1095,60,NA,TRUE,mosipadmin,now()
```

Comment:
how will we handle scenario, where this is installed on top of existing system

Reply:
There're are two ways to solve this IMO.

1. A pre-deploy script can be written for this for a primary deployment & a longer term solution could be via the release upgrade scripts.
2. The correct way to get this done might involve some work & as per the best practices it has to be done via INJICERT-469.